static-assets/scripts/host.js

```diff
@@ -634,7 +633,6 @@
   }
 
   function selectStudioContent(site, url) {
-    CStudioAuthoring.ContextualNav.WcmActiveContent?.disableNav(true);
```

Comment:
I think these may be covering other scenarios and we should leave them as long as they cause no trouble.